Automattic/wp-calypso
# Scan: Add unignore threat capabilities

Related to #
- https://github.com/Automattic/jetpack-scan-team/issues/1312

## Proposed Changes
Adds the ability to unignore threats that were previously ignored.

## Why are these changes being made?

* To provide a new feature and to bring parity to the various access points of the scan upgrade.

## Testing Instructions

* Checkout this branch and start your dev environment
* Proceed to `http://calypso.localhost:3000/scan/<site>` for a testing site with existing threats
* Ignore several threats and proceed to History and filter by `Ignored`
* Select `Unignore threat` and verify that the threat is added back to the current threats list
* Check `vp-global` to ensure the `vp_scan_results` entry was updated as expected
